src/value/owned.rs

```diff
@@ -335,3 +335,118 @@ impl Element for OwnedElement {
         }
     }
 }
+
+#[cfg(test)]
+mod value_tests {
+    use crate::result::IonResult;
+    use crate::types::decimal::Decimal;
+    use crate::types::timestamp::Timestamp;
+    use crate::value::owned::{OwnedElement, OwnedSequence, OwnedSymbolToken, OwnedValue};
+    use crate::value::{AnyInt, Element, IntAccess, SymbolToken};
+
+    fn extract_text<T: SymbolToken>(tok: &T) -> String {
+        tok.text().unwrap().into()
+    }
```

Comment:
Curious why you need to convert `&str` to `String`?

Reply:
No specific reason. I think I tried to replicate from the doc tests of `SymbolToken`. But I think using `&str ` would save an extra step of converting to `String` .